dv/verilator/sonata_verilator_lint.vlt

```diff
@@ -40,3 +40,5 @@ lint_off -rule WIDTH  -file "*uartdpi.sv"
 lint_off -rule UNUSED -file "*uartdpi.sv"
 
 lint_off -rule WIDTH -file "*uart_core.sv"
+
+lint_off -rule UNUSED -file "*/rtl/system/rv_timer.sv" -match "*'timer_addr_i'[31:16]*"
```

Comment:
I prefer labelling these as unused in the rtl itself.

Reply:
I don't know of a way to use `-match` in verilog comments. I like the match syntax because it is very specific about which indices are allowed to be unused.

Reply:
Something like this is what I mean: https://github.com/lowRISC/sonata-system/blob/7ab9775dac54651477d41007936c0658b5f1b0d0/rtl/system/spi_top.sv#L128

Reply:
Ah that is nicer, I'll do that